client/ionic/src/pages/Triage.tsx

```diff
@@ -4,7 +4,9 @@ import TopNav from '../components/TopNav';
 import * as Survey from 'survey-react';
 import 'survey-react/survey.css';
 import triage from '../surveys/triage.json';
+import countries from '../surveys/countries.json';
 import 'tachyons';
+import '../theme/custom.css';
```

Comment:
I propose moving this to `triage.module.css`.
This CSS is only relevant for this page.

Reply:
did `triage.css`